core-bundle/src/Resources/contao/forms/FormFileUpload.php

```diff
@@ -74,6 +74,16 @@ public function __set($strKey, $varValue)
 				}
 				break;
 
+			case 'extensions':
+				if ($varValue)
+				{
+					$accept = preg_replace("/\s/", "", $varValue);
+					$accept = preg_replace("/[^,]+/", ".$0", $accept);
```

Comment:
I’m not sure if we should use regular expressions here. How about something like this: https://github.com/contao/contao/blob/198adf77b8d3d50dd8a664062814bdce216d9025/core-bundle/src/Resources/contao/classes/DropZone.php#L31

Reply:
I agree, changed it.